src/main/java/com/github/flaxsearch/resources/PostingsResource.java

```diff
@@ -0,0 +1,79 @@
+package com.github.flaxsearch.resources;
+/*
+ *   Copyright (c) 2016 Lemur Consulting Ltd.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+import com.github.flaxsearch.util.ReaderManager;
+import org.apache.lucene.index.*;
+import org.apache.lucene.util.Bits;
+import org.apache.lucene.util.BytesRef;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.*;
+
+@Path("/postings/{field}")
+@Produces(MediaType.APPLICATION_JSON)
```

Comment:
I think this would be neater done as a per-term response, with a path of /postings/{field}/{term}